MODIFY.md

```diff
@@ -0,0 +1,97 @@
+# Modifying
```

Comment:
you dont cover modify LLMs?

Also really weird structure. you vectorstore/backend/frontend all on same level

some of those are directory, some of those are components